js/main.js

```diff
@@ -141,6 +141,8 @@ var ThreeScene = {
 
         console.log('Files percent loaded', _this.percentLoaded);
 
+        $('#loading-percent').html(_this.percentLoaded + '%');
```

Comment:
This number can have an infinite number of decimal places. You want to round down or?